internal/graphicsdriver/opengl/gl/package_darwin.go

```diff
@@ -0,0 +1,686 @@
+}
+
+func CompileShader(shader uint32) {
+	purego.SyscallN(gpCompileShader, uintptr(shader))
+}
+
+func CreateProgram() uint32 {
+	ret, _, _ := purego.SyscallN(gpCreateProgram)
+	return (uint32)(ret)
+}
+
+func CreateShader(xtype uint32) uint32 {
+	ret, _, _ := purego.SyscallN(gpCreateShader, uintptr(xtype))
+	return (uint32)(ret)
+}
+
+func DeleteBuffers(n int32, buffers *uint32) {
+	purego.SyscallN(gpDeleteBuffers, uintptr(n), uintptr(unsafe.Pointer(buffers)))
+}
+
+func DeleteFramebuffersEXT(n int32, framebuffers *uint32) {
+	purego.SyscallN(gpDeleteFramebuffersEXT, uintptr(n), uintptr(unsafe.Pointer(framebuffers)))
+}
+
+func DeleteProgram(program uint32) {
+	purego.SyscallN(gpDeleteProgram, uintptr(program))
+}
+
+func DeleteRenderbuffersEXT(n int32, renderbuffers *uint32) {
+	purego.SyscallN(gpDeleteRenderbuffersEXT, uintptr(n), uintptr(unsafe.Pointer(renderbuffers)))
+}
+
+func DeleteShader(shader uint32) {
+	purego.SyscallN(gpDeleteShader, uintptr(shader))
+}
+
+func DeleteTextures(n int32, textures *uint32) {
+	purego.SyscallN(gpDeleteTextures, uintptr(n), uintptr(unsafe.Pointer(textures)))
+}
+
+func Disable(cap uint32) {
+	purego.SyscallN(gpDisable, uintptr(cap))
+}
+
+func DisableVertexAttribArray(index uint32) {
+	purego.SyscallN(gpDisableVertexAttribArray, uintptr(index))
+}
+
+func DrawElements(mode uint32, count int32, xtype uint32, indices uintptr) {
+	purego.SyscallN(gpDrawElements, uintptr(mode), uintptr(count), uintptr(xtype), uintptr(indices))
+}
+
+func Enable(cap uint32) {
+	purego.SyscallN(gpEnable, uintptr(cap))
+}
+
+func EnableVertexAttribArray(index uint32) {
+	purego.SyscallN(gpEnableVertexAttribArray, uintptr(index))
+}
+
+func Flush() {
+	purego.SyscallN(gpFlush)
+}
+
+func FramebufferRenderbufferEXT(target uint32, attachment uint32, renderbuffertarget uint32, renderbuffer uint32) {
+	purego.SyscallN(gpFramebufferRenderbufferEXT, uintptr(target), uintptr(attachment), uintptr(renderbuffertarget), uintptr(renderbuffer))
+}
+
+func FramebufferTexture2DEXT(target uint32, attachment uint32, textarget uint32, texture uint32, level int32) {
+	purego.SyscallN(gpFramebufferTexture2DEXT, uintptr(target), uintptr(attachment), uintptr(textarget), uintptr(texture), uintptr(level))
+}
+
+func GenBuffers(n int32, buffers *uint32) {
+	purego.SyscallN(gpGenBuffers, uintptr(n), uintptr(unsafe.Pointer(buffers)))
+}
+
+func GenFramebuffersEXT(n int32, framebuffers *uint32) {
+	purego.SyscallN(gpGenFramebuffersEXT, uintptr(n), uintptr(unsafe.Pointer(framebuffers)))
+}
+
+func GenRenderbuffersEXT(n int32, renderbuffers *uint32) {
+	purego.SyscallN(gpGenRenderbuffersEXT, uintptr(n), uintptr(unsafe.Pointer(renderbuffers)))
+}
+
+func GenTextures(n int32, textures *uint32) {
+	purego.SyscallN(gpGenTextures, uintptr(n), uintptr(unsafe.Pointer(textures)))
+}
+
+func GetBufferSubData(target uint32, offset int, size int, data unsafe.Pointer) {
```

Comment:
Remove GetBufferSubData (this was removed at https://github.com/hajimehoshi/ebiten/commit/3745c0a2e0ffe0e65dfe746188402a4c22ff5f4a)